grits/tests/test_utils.py

```diff
@@ -16,3 +19,49 @@ def test_get_hydrogen():
 
     methane = load("C", smiles=True)
     assert isinstance(get_hydrogen(methane, methane[0]), Compound)
+
+
+def test_get_com():
+    import numpy as np
+
+    from grits.utils import get_com
+
+    pos_arr = np.zeros([4, 3])
+    mass_arr = np.ones(4)
+
+    # square
+    pos_arr[0] = np.array([0, 1, 1])
+    pos_arr[1] = np.array([0, 1, -1])
+    pos_arr[2] = np.array([0, -1, 1])
+    pos_arr[3] = np.array([0, -1, -1])
+    assert np.allclose(get_com(pos_arr, mass_arr), np.zeros(3))
+
+    # line
+    pos_arr[0] = np.array([0, 2, 2])
+    pos_arr[1] = np.array([0, 1, 1])
+    pos_arr[2] = np.array([0, -2, -2])
+    pos_arr[3] = np.array([0, -1, -1])
+    assert np.allclose(get_com(pos_arr, mass_arr), np.zeros(3))
+
+    # biased line
+    mass_arr[0] = 5
+    mass_arr[1] = 10
+    com = get_com(pos_arr, mass_arr)
+    assert com[1] > 0 and com[2] > 0
+
+
+class TestAnisoUtils(BaseTest):
+    def test_get_heavy_atoms(self, butane_gsd):
+        import gsd.hoomd
+
+        from grits.utils import get_heavy_atoms
+
+        with butane_gsd as f:
+            assert type(f) == gsd.hoomd.HOOMDTrajectory
+            frame = f[0]
+            particles = frame.particles
+            heavy_positions, heavy_masses = get_heavy_atoms(particles)
+            # 4 carbons -> 4 entries
+            assert len(heavy_masses) == len(heavy_positions) == 4
+            # approximate mass
+            assert round(sum(heavy_masses)) == 48
```

Comment:
I think we should make sure hydrogen masses are being included in the bead mass. Butane g/mol is 58, so this is missing the 10 hydrogens. This was an issue in #52  and should be addressed by #53

Reply:
Oops, I misread this test. But I pushed a commit that tests the mass in the CG gsd files written out as well.